misc/helm-charts/testing/README.md

```diff
@@ -56,32 +56,37 @@ kubectl get nodes --show-labels
 
 ## Metrics service
 
-The metrics service is required for the `environmentd` pod to function correctly.
+The metrics service is required for the `environmentd` pod to function correctly. For more information, see the [Metrics Server documentation](https://github.com/kubernetes-sigs/metrics-server/blob/master/README.md).
```

Comment:
Q: If this is required for the environmentd to function correctly, should we update the Chart's values.yaml file to have the observability.podMetrics.enabled be true?  Or mention that we should set it to true?

Reply:
Ah yes, good point! This was actually true in the past, now if the `observability` is disabled in the Helm values file, envd will still work. It will only fail if the `observability` is enabled but the metrics service is missing. I've added a bit of clarification for this. Let me know what you think!

Reply:
Looks great! Thanks for the clarification.